docs/Community/sig-release/charter.md

```diff
@@ -0,0 +1,47 @@
+SIG Release Charter
```

Comment:
that's an interesting question -- I didn't intend to _change_ the release (approval) process in this patch. 

I'll leave the docs in their current state, even if as you point out it is inconsistent, so that it can be discussed and corrected without me having to guess at the process